src/sentry/incidents/logic.py

```diff
@@ -1103,12 +1107,17 @@ def update_alert_rule_trigger_action(
         if type in AlertRuleTriggerAction.INTEGRATION_TYPES:
             integration = updated_fields.get("integration", trigger_action.integration)
             organization = trigger_action.alert_rule_trigger.alert_rule.organization
+
             channel_id = get_alert_rule_trigger_action_integration_object_id(
```

Comment:
so I now hate this variable name but am struggling to find a better one. For PD I'm returning a service because I need to pull out the service ID and name (and expect OpsGenie to be similar).